webapp/public/locales/fr/main.json

```diff
@@ -481,6 +485,8 @@
   "study.modelization.clusters.matrix.common": "Common",
   "study.modelization.clusters.matrix.tsGen": "TS generator",
   "study.modelization.clusters.matrix.timeSeries": "Séries temporelles",
+  "study.modelization.clusters.matrix.fuelCost": "Fuel Cost",
```

Comment:
Not translated

Reply:
same it's not provided if a translation is needed we will add it later